pkgs/by-name/hm/hmrc-paye-tools/package.nix

```diff
@@ -0,0 +1,84 @@
+{ lib
+, stdenv
+, fetchurl
+, buildFHSEnv
+, unzip
+}:
+
+let
+  installer = stdenv.mkDerivation rec {
+    name = "hmrc-paye-tools-installer";
+    version = "24.1.24086.542";
+
+    src = fetchurl {
+      url = "https://www.gov.uk/government/uploads/uploaded/hmrc/payetools-rti-24.1.24086.542-linux.zip";
+      sha256 = "QOW6Loqg001AcqWX/TOH6wvI2uAY4qNyFvQzCVEe8VU=";
+    };
+
+    buildInputs = [ unzip ];
+
+    phases = [ "buildPhase" ];
+
+    buildPhase = ''
+      mkdir -p $out/bin
+      unzip $src -d $out/bin
+      chmod +x $out/bin/*
+    '';
+  };
+
+  installerFHS = buildFHSEnv {
+    name = "hmrc-paye-tools-installer-fhs";
+    targetPkgs = pkgs: (with pkgs; [
+      glibc
+    ]);
+    runScript = "${installer}/bin/payetools-rti-24.1.24086.542-linux";
+  };
+
+  package = stdenv.mkDerivation {
+    name = "hmrc-paye-tools";
+    version = "24.1.24086.542";
+
+    phases = [ "buildPhase" ];
```

Comment:
Do not set `phases` directly. Use `dontUnpack`, `dontInstall`, `dontFixup`, etc.